### PR TITLE
A path to fixing Nano loading issues.

### PR DESCRIPTION
**Pull request by PJB3005**
Makes template loading time out instead of hang forever if necessary.
This will make it easier to diagnose the problem causing load failures
with Nano.

Also made all of Nano's .js files use tabs.

[system] [ui]

### CODE REVIEW

File: nano/js/nano_state_manager.js [hunk 1]
```diff
@@ -74,68 +74,68 @@ NanoStateManager = function ()
...
 	};
 
 	// This function does the update by calling the methods on the current state
 	var doUpdate = function (data)
 	{
-        if (_currentState == null)
-        {
-            return;
-        }
+		if (_currentState == null)
```
Comment (ShiftyRail) [inline nano/js/nano_state_manager.js:91]: Got indented 1 tab forwards

File: nano/js/nano_state_default.js [hunk 2]
```diff
@@ -4,11 +4,11 @@ var NanoStateDefault = new NanoStateDefaultClass();
 
 function NanoStateDefaultClass() {
 
-    this.key = 'default';
+	this.key = 'default';
```
Comment (ShiftyRail) [inline nano/js/nano_state_default.js:7]: Got indented one tab forwards (other lines lower down the file as well)

File: nano/js/nano_state_manager.js [hunk 3]
```diff
@@ -194,31 +194,30 @@ NanoStateManager = function ()
 		setCurrentState: function (stateKey)
 		{
 			if (typeof stateKey == 'undefined' || !stateKey) {
-				alert('ERROR: No state key was passed!');				
-                return false;
-            }
+				alert('ERROR: No state key was passed!');
+				return false;
```
Comment (ShiftyRail) [inline nano/js/nano_state_manager.js:198]: Indented a few tabs forwards

**Review by ShiftyRail — COMMENTED** [review]
Very hard to understand what actually got changed tbh, is it just moving stuff a few tabs forwards ?

File: nano/js/nano_state_default.js (hunk 2 above)
Comment (PJB3005) [reply nano/js/nano_state_default.js:7]: No, it didn't. This is because GitHub displays tabs as 8 spaces wide.

**Comment by PJB3005** [comment]
> Very hard to understand what actually got changed tbh, is it just moving stuff a few tabs forwards ?

Read the PR body.

Also turn on whitespace hiding to actually read the PR diff sanely.

image

**Pull request merged**